COPYRIGHT.txt

```diff
@@ -126,6 +126,11 @@ Copyright: 2018, Eric Lasota
  2018, Microsoft Corp.
 License: Expat
 
+Files: ./thirdparty/doctest/
+Comment: doctest
+Copyright: 2019, Viktor Kirilov
```

Comment:
2020.

Reply:
Er no, nevermind. License file states "2016-2019" so that's what we should use: https://github.com/onqtam/doctest/blob/2.4.0/LICENSE.txt
```suggestion
Copyright: 2016-2019, Viktor Kirilov
```